Fix the depedencies in pyproject.toml to match the ones in the package
--- pgsu-0.2.4/setup.cfg.orig	2024-05-27 18:14:54.068433566 +0200
+++ pgsu-0.2.4/setup.cfg	2023-07-11 06:17:02.484777500 +0200
@@ -27,7 +27,7 @@
 packages = find:
 install_requires = 
 	click
-	psycopg2-binary>=2.8.3
+	psycopg2>=2.8.3
 python_requires = ~=3.6
 
 [options.packages.find]
